elements/pfe-sass/extends/_typography.scss

```diff
@@ -0,0 +1,71 @@
+%pfe-heading-3xl {
```

Comment:
```suggestion
%pfe-heading--3xl {
```

BEM suggests modifiers be separated with `--`

Reply:
updated!